src/cmdlib.sh

```diff
@@ -402,6 +402,7 @@ runvm() {
 set -xeuo pipefail
 export PATH=/usr/sbin:$PATH
 workdir=${workdir}
+builddir=$(pwd)
```

Comment:
Let's rename this e.g. `host_pwd=$(pwd)` ?

IOW, I don't think we should think of this as a `builddir` since we don't always have one. Whereas if we just reframe it as "make supermin command run in the same dir as where we were", that achieves the same goal and seems a less surprising behaviour than always `cd`'ing to the workdir.

Let's maybe also add a check that the `host_pwd` is under `workdir` and just error out otherwise? That way we ensure that we end up under the workdir mount.